core/src/main/scala/com/softwaremill/react/kafka2/DummyConsumer.scala

```diff
@@ -14,14 +15,17 @@ object Streams {
     case Failure(ex) =>
       println("Stream finished with error")
       ex.printStackTrace()
-      as.shutdown()
+      import scala.concurrent.duration._
```

Comment:
as for me we should keep same semantic. Semantic of this was terminate actor system, but not waiting for termination. Lets keep existing semantic.